deployments/datahub/images/default/environment.yml

```diff
@@ -99,9 +99,12 @@ dependencies:
   # - -r /tmp/infra-requirements.txt
   # Econ 148, Spring 2023 https://github.com/berkeley-dsep-infra/datahub/issues/4093
   - pycountry-convert==0.7.2
-  # error converting to PDF https://github.com/berkeley-dsep-infra/datahub/issues/5062
+  # Options to convert notebooks to pdf; error converting to PDF https://github.com/berkeley-dsep-infra/datahub/issues/5062
   - nbconvert[webpdf]
```

Comment:
try changing this to `nbconvert[webpdf]==7.6.0`

Reply:
will do that now!